coderd/provisionerdaemons.go

```diff
@@ -110,6 +110,13 @@ type workspaceProvisionJob struct {
 	DryRun           bool      `json:"dry_run"`
 }
 
+// The input for a "template_version_plan" job.
+type templateVersionPlanJob struct {
+	TemplateVersionID uuid.UUID                 `json:"template_version_id"`
+	WorkspaceName     string                    `json:"workspace_name"`
```

Comment:
Is this just visual for logs?

Reply:
No this runs the plan with the desired name. Optional, defaults to `plan` if empty